hack/operatorhub/templates/package.tpl

```diff
@@ -1,5 +1,5 @@
 channels:
-- currentCSV: {{ .PackageName }}.v{{ .NewVersion }}
+- currentCSV: {{ .PackageName }}.{{ .NewVersion }}
```

Comment:
I think we need to keep the `v`.
```suggestion
- currentCSV: {{ .PackageName }}.v{{ .NewVersion }}
```

Otherwise, I get:

```
> OHUB_TAG=2.6.1 hack/operatorhub/bin/operatorhub bundle generate --dir=.
2023/02/10 15:19:38 Generating operator bundle image for publishing
2023/02/10 15:19:38 ⅹ
Error: during opm generate: error interpreting channels, please manually input channels instead
2023/02/10 15:19:38 failed to run command: during opm generate: error interpreting channels, please manually input channels instead
```

Because we use the `v`prefix to name the CSV:
```
# templates/csv.tpl
apiVersion: operators.coreos.com/v1alpha1
kind: ClusterServiceVersion
metadata:
  name: {{ .PackageName }}.v{{ .NewVersion }}
```